lib/flutter_web_auth.dart

```diff
@@ -29,9 +30,13 @@ class FlutterWebAuth {
   ///
   /// [callbackUrlScheme] should be a string specifying the scheme of the url that the page will redirect to upon successful authentication.
   /// [preferEphemeral] if this is specified as `true`, an ephemeral web browser session will be used where possible (`FLAG_ACTIVITY_NO_HISTORY` on Android, `prefersEphemeralWebBrowserSession` on iOS/macOS)
-  static Future<String> authenticate({required String url, required String callbackUrlScheme, bool? preferEphemeral}) async {
-    WidgetsBinding.instance?.removeObserver(_resumedObserver); // safety measure so we never add this observer twice
-    WidgetsBinding.instance?.addObserver(_resumedObserver);
+  static Future<String> authenticate(
+      {required String url,
+      required String callbackUrlScheme,
+      bool? preferEphemeral}) async {
+    WidgetsBinding.instance.removeObserver(
+        _resumedObserver); // safety measure so we never add this observer twice
+    WidgetsBinding.instance.addObserver(_resumedObserver);
```

Comment:
I guess it would help to not change the formatting of the code, so that it's easier to spot the actual changes.

Reply:
Understandable. Might as well rip off the band aid now, though.